codeception.yml

```diff
@@ -0,0 +1,17 @@
+actor: Tester
+paths:
+    tests: tests
+    log: tests/_output
```

Comment:
Most everything in this branch was generated by running `vendor/bin/codecept bootstrap`, and I left most of it as is; I deleted unnecessary directories and files that we wouldn't need, such as the integration test suite.

Underscores and the directory names are what Codeception defaults to when running the bootstrap command.

If I recall correctly, `_data` was intended to use sql dumps.

It might be best to review the commits in order to give better context.